packages/new-editor/src/components/editor/variables/detail/Variable.tsx

```diff
@@ -2,15 +2,20 @@ import { Fieldset, Input, SimpleSelect, Textarea, ToolbarTitle } from '@axonivy/
 import { metadataOptions, type Variable } from '../../data/Variable';
 
 type VariableProps = {
-  variable: Variable;
+  variable: Variable | undefined;
```

Comment:
instead of `| undefined` you can also just add a ? do the variable: `variable?: Variable`